src/Utils.cpp

```diff
@@ -143,6 +143,7 @@ obs_data_t* Utils::GetSceneItemData(obs_sceneitem_t* item) {
 	obs_data_set_double(data, "y", pos.y);
 	obs_data_set_int(data, "source_cx", (int)item_width);
 	obs_data_set_int(data, "source_cy", (int)item_height);
+	obs_data_set_int(data, "alignment", (int)obs_sceneitem_get_alignment(item));
```

Comment:
Ah. It's already in there. The entire point of this PR is to make the same info available when you switch scenes; the event payload includes all of this info from GetSceneItemData, with all the rest of the positional info, but omits the alignment. That means that it's possible to completely render a scene framework from the event payload but ONLY by assuming that everything is aligned top-left. To be able to cope with alignment (which is important, since it changes the interpretation of the x and y position), you currently need to GetSceneItemProperties individually for every single item in the scene.

Reply:
Can this be considered again please? Without knowing the alignment, it's almost useless to know the position. Currently, to make use of the position in any meaningful way, you have to query every individual item to find out its alignment.

Reply:
Sure! I'm fine with that addition.